<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBCreation</class>
 <widget class="QDialog" name="DBCreation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton 
{
text-align: left;
padding: 3px;
padding-left: 5px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_password">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="maxLength">
      <number>16</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_name">
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="text">
      <string>Plain account name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_confirm">
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="maxLength">
      <number>16</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_confirm">
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="text">
      <string>Re-enter password</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QLabel" name="label_error">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid red;
padding: 5px;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="label_logo">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_password">
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="text">
      <string>Choose a Password</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>22</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #7da10e;</string>
     </property>
     <widget class="QLabel" name="label_available">
      <property name="geometry">
       <rect>
        <x>29</x>
        <y>1</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Available</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_available_icon">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>4</y>
        <width>14</width>
        <height>14</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_done">
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>DONE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>CANCEL</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_login">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="maxLength">
      <number>16</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_login</tabstop>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>lineEdit_confirm</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
